resources/definitions/diy220.def.json

```diff
@@ -0,0 +1,89 @@
+{
+    "version": 2,
+    "name": "Diytech 220",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "Venkatkamesh",
+        "manufacturer": "Sri Vignan Technologies",
+        "weight": 3,
+        "file_formats": "text/x-gcode",
+        "platform": "ultimaker3_platform.obj",
```

Comment:
Uh? Why is this machine using the same model as the Ultimaker?

Reply:
Please make a single comment on same def 
This definition is since 2.7 update
Thank you